app/components/applications-app-services/function-app.tf

```
@@ -279,3 +278,29 @@ resource "azurerm_servicebus_subscription_rule" "nsip_advice_topic_subscription_
     }
   }
 }
+
+# nsip-service-user
+resource "azurerm_servicebus_subscription" "nsip_service_user_topic_subscription" {
+  count = var.feature_back_office_subscriber_enabled ? 1 : 0
+
+  name               = "applications-nsip-service_user"

```

```suggestion
  name               = "applications-nsip-service-user"

```

maybe a dash for consistency with other resources?